Tests/Marketplace/upload_packs.py

```diff
@@ -1060,12 +1071,35 @@ def main():
 
         # in case that pack already exist at cloud storage path and in index, don't show that the pack was changed
         if skipped_upload and exists_in_index:
+            logging.info(f"{pack.name} pack status is {PackStatus.PACK_ALREADY_EXISTS.name}")
             pack.status = PackStatus.PACK_ALREADY_EXISTS.name
             pack.cleanup()
             continue
 
         pack.status = PackStatus.SUCCESS.name
 
+    logging.info(f"packs_missing_dependencies: {packs_missing_dependencies}")
+
+    # will go over all the packs what was marked as missing dependencies and will update them with the new index.json
+    for pack in packs_missing_dependencies:
+
+        task_status = pack.format_metadata(user_metadata, index_folder_path, packs_dependencies_mapping, build_number,
+                                           current_commit_hash, pack_was_modified, statistics_handler, pack_names, True)
+
+        if not task_status:
+            pack.status = PackStatus.FAILED_METADATA_PARSING.name
+            pack.cleanup()
+            continue
```

Comment:
Please add a new status for this task, we need to be able to know what is the failure status of the pack